decidim-conferences/lib/decidim/conferences/admin_engine.rb

```diff
@@ -64,6 +64,7 @@ class AdminEngine < ::Rails::Engine
             resources :imports, only: [:new, :create] do
               get :example, on: :collection
             end
+            resources :reminders, only: [:new, :create]
```

Comment:
This is missing in `decidim-assemblies/lib/decidim/assemblies/admin_engine.rb`, so it's giving an exception `undefined method new_component_reminder_path for #<Module:0x00005568a4b596b0>`. Can you add it 🙏🏽 ?